bindings/python/examples/saveCCBToFile/readme.md

```diff
@@ -0,0 +1,13 @@
+# First frame Example
+
+### Overview
+This example saves the hardwares's CCB to a file
```

Comment:
Change

**This example saves the hardwares's CCB to a file**

to

**This tool saves the CCB of the imager's module to a file.**